qgis2web/leafletLayerScripts.py

```diff
@@ -183,9 +180,9 @@ def getLabels(layer, safeLayerName, outputProjectFileName, vts, vtLabels):
             fontItalic = font.italic()
             fontBold = font.bold()
             fontColor = text.color().name()
-            fontUnderline = font.underline()
-            xOffset = palyr.xOffset
-            yOffset = palyr.yOffset
+            # fontUnderline = font.underline()
```

Comment:
Are these implemented in a different way now? I'll find out.

Reply:
not it was just complaining that these were not actually used anywhere.